File: wear/src/main/java/com/thewizrd/simplewear/controls/SleepTimerViewModel.java
package com.thewizrd.simplewear.controls;

import androidx.lifecycle.ViewModel;

public class SleepTimerViewModel extends ViewModel {
    public static final int DEFAULT_TIME_MIN = 5;
    public static final int MAX_TIME_IN_MINS = 120;

    private int progressTimeInMins = DEFAULT_TIME_MIN;

    public int getProgressTimeInMins() {
        return progressTimeInMins;
    }

    public void setProgressTimeInMins(int progressTimeInMins) {
        this.progressTimeInMins = progressTimeInMins;
    }

    @Override
    protected void onCleared() {
        super.onCleared();
        progressTimeInMins = DEFAULT_TIME_MIN;
    }
}

File: wear/src/main/java/com/thewizrd/simplewear/controls/SleepTimerViewModel.kt
package com.thewizrd.simplewear.controls

import androidx.lifecycle.ViewModel

class SleepTimerViewModel : ViewModel() {
    companion object {
        const val DEFAULT_TIME_MIN = 5
        const val MAX_TIME_IN_MINS = 120
    }

    var progressTimeInMins = DEFAULT_TIME_MIN

    override fun onCleared() {
        super.onCleared()
        progressTimeInMins = DEFAULT_TIME_MIN
    }
}